.ci/config/matrices.yaml

```diff
@@ -90,6 +90,7 @@ jdk-ce-stages:
   - 'platform-jdk-openjdk-jdk-12-latest'
   - 'platform-jdk-openjdk-jdk-11-latest'
   - 'platform-jdk-openjdk-jdk-8-latest'
+  - 'platform-jdk-jdk-15-latest'
```

Comment:
Shouldn't there be a `platform-jdk-openjdk-jdk-15-latest` stage as well?

Reply:
Well, INFRA only added an Oracle JDK 15 so far. But yes, I can ping them regarding that. 👍

Reply:
INFRA added the environment, I added a new stage 😃